### ISSUE TITLE
Aggregate Block timestamp data with Transaction data

### ISSUE THREAD

**Issue by ghoti143** [EOSEssentials/EOSTracker-API#8]
In the TransactionController, the transactionsAction should aggregate data from the Block collection to enable the timestamp from the transaction's containing block to be returned with the transaction data.

Here is an example mongo query that can be used as a starting point:

```
//this requires at least Mongo 3.2

db.Transactions.aggregate([
{$lookup: {
  from: "Blocks",
  localField: "block_id",
  foreignField: "block_id",
  as: "block_docs"
}},
{$unwind: '$block_docs'},
{$project: {
    timestamp: '$block_docs.timestamp',
    transaction_id: 1,
    sequence_num: 1,
    block_id: 1,
    ref_block_num: 1,
    ref_block_prefix: 1,
    expiration: 1,
    signatures: 1,
    actions: 1,
    createdAt: 1
}}
])
```

There's probably a more concise way to say "select the timestamp field from Block record and all of the fields from the Transaction record."

**Comment by ghoti143** [comment]
I spent a bit of time looking into making this more performant.  One factor is that the sort operation should come at the top of the pipeline before doing the lookup.  But if the sort is before the lookup, only the main Document's fields can be used for sorting.  So it looks like sorting on the block's timestamp is going to be pretty difficult.

As an alternative, I discovered the sequence_num column, which according to [DB Schema Defintion - EOS API](https://docs.google.com/document/d/1E6BQVX1E8jsCzJIpFWV_B3hfA7Br6iiFlUiWZJpJ3zM), is the "Ordinal sequence of this transaction for sorting."  I'm not sure if that field is the sequence across all transactions or just internal to the block.  There's also the ref_block_num column, which could also be used since the block height is always increasing.
